### PR TITLE
Feature validation enhancements 2023

### PR DESCRIPTION
**Pull request by samshara**
Addresses 

- [x] https://github.com/mapswipe/mapswipe/issues/789
- [x] https://github.com/mapswipe/mapswipe/issues/788
- [x] https://github.com/mapswipe/mapswipe/issues/786
- [x] https://github.com/mapswipe/mapswipe/issues/791
- [x] https://github.com/mapswipe/mapswipe/issues/792
- [x] https://github.com/mapswipe/mapswipe/issues/790
- [x] https://github.com/mapswipe/mapswipe/issues/787
- [x] https://github.com/mapswipe/mapswipe/issues/798
### Additional cross application linked issues
- [x] https://github.com/mapswipe/python-mapswipe-workers/issues/822
- [x] https://github.com/mapswipe/python-mapswipe-workers/issues/842
- [x] https://github.com/mapswipe/python-mapswipe-workers/issues/830

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

File: src/shared/common/InformationPage.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                                    <Image
+                                        style={styles.introImage}
+                                        source={block.image}
+                                    />
+                                ) : (
+                                    <Image
+                                        style={styles.introImage}
+                                        src={block.image}
+                                    />
+                                )}
```
Comment (tnagorra) [inline src/shared/common/InformationPage.js:100]: There is no difference between second and third expression

File: src/shared/common/Tutorial/index.js [hunk 2]
```diff
@@ -84,6 +97,9 @@ export default class TutorialBox extends React.Component<Props, State> {
         const {
             content: { icon },
         } = this.props;
+
+        console.warn('icon', icon);
```
Comment (tnagorra) [inline src/shared/common/Tutorial/index.js:101]: Let's remove the warning

File: src/shared/views/BuildingFootprint/TutorialIntro.js [hunk 3]
```diff
@@ -62,86 +86,110 @@ const styles = StyleSheet.create({
...
-        <View style={styles.background}>
-            <View style={styles.screenWidth}>
-                <ScrollView style={styles.container}>
-                    <Text style={styles.header}>{t('letsLearnHowToMap')}</Text>
+    const {
+        t,
+        informationPages: informationPagesFromProps,
+        customOptions,
+    } = props;
+    const fallbackInformationPage: ProjectInformation = [
```
Comment (tnagorra) [inline src/shared/views/BuildingFootprint/TutorialIntro.js:110]: Let's move this outside the function

File: src/shared/views/BuildingFootprint/index.js [hunk 4]
```diff
@@ -52,32 +82,41 @@ class _BuildingFootprintScreen extends React.Component<Props> {
     }
 
     /* eslint-disable global-require */
-    getNormalHelpContent = () => {
+    getNormalHelpContent = (_, customOptions: Option[]) => {
         const { t } = this.props;
         return (
             <>
-                <Text style={styles.header}>{t('howToContribute')}</Text>
+                <Text style={styles.header}>TThis is good</Text>
```
Comment (tnagorra) [inline src/shared/views/BuildingFootprint/index.js:89]: TThis is not good.

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Comment by frozenhelium** [comment]
@laurentS This is ready to be be deployed! Let's go ahead with the release! CC: @danbjoseph

**Review by tnagorra — APPROVED** [review]

**Pull request merged**